File: plusmall-operator-web/src/main/webapp/js/service/goodsAuditService.js
app.service('goodsAuditService',function ($http) {

    //查找
    this.search=function (pageNum, pageSize, searchEntity) {
        return $http.post('../goodsAudit/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    }

    //更新商品状态
    this.updateStatus=function (ids, status) {
        return $http.get('../goodsAudit/updateStatus.do?ids='+ids+'&status='+status);
    }
})

File: plusmall-operator-interface/src/main/java/com/plusmall/operator/GoodsAuditService.java
package com.plusmall.operator;

import com.plusmall.commons.PageResult;
import com.plusmall.model.TbGoods;
import com.plusmall.model.TbItem;

import java.util.List;

/**
 * @Description:
 */
public interface GoodsAuditService {
	public PageResult search(int pageNum, int pageSize, TbGoods tbGoods);
	public void updateStatus(Long[] ids,String status) throws NullPointerException;
	public void delete(Long[] ids) throws NullPointerException;
	public List<TbItem> findItemListByGoodsIdAndStatus(Long[] goodsIds, String status ) throws NullPointerException;
}
